recipes/requests-async/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set name = "requests-async" %}
+{% set version = "0.6.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3bfdd7f4b4d24ae86c0382c6da90ba704e9629b9d803f6f79d2feb39a480f3c0
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - http3 ==0.1.*
+    - python >=3.6
+    - requests ==2.*
+
+test:
+  imports:
+    - requests_async
+
+about:
+  home: https://github.com/encode/requests-async
+  license: BSD
```

Comment:
Could you please double check the license? I think that should be ``Apache-2.0``

Reply:
https://github.com/encode/requests-async/blob/master/LICENSE.md

Reply:
Indeed, the metadata on PyPI is incorrect.

Reply:
https://github.com/encode/requests-async/pull/56